### DISCUSSION TITLE
Review of ancestral graphs from a theoretical perspective

### DISCUSSION THREAD

**Discussion by jeromekelleher** [tskit-dev/what-is-an-arg-paper#13] [General]
One of the key goals of this paper is to clarify the meaning of the term ARG, and a crucial part of the confusion that exists is that two different communities simultaneously use the term to mean different things. Roughly, the theoretical community understand the ARG to mean an ancestral *process* and the (for want of a better word) empirical community think of it as the *outcome* of such a process.

The idea of this section is to review the literature on ancestral graphs from the theoretical perspective. This gives us a place to systematically set out the development of the ideas from the theoretical perspective, including the Ancestral Selection Graph, the Ancestral Gene Transfer Graph, and any other "ancestral graphs" that we think worth mentioning. I think it's worthwhile giving a good overview here, as it will be helpful to the "empiricists" to realise that there is this large parallel literature on ancestral-graphs-as-processes.

A good first step here would be to gather together all the references we think are relevant into an enumeration. We can turn this into a narrative later. #12 added some initial work on this.

@JereKoskela you're probably in a good position to do this: would you mind doing an "ancestral graph" brain dump in this section at some point please?

**Comment by hyanwong** [comment]
Re-reading the Griffiths papers, it seems that they use the graph to mean *both* the process and the outcome: indeed, I feel this might be the root of the confusion, because the two are not clearly distinguished there.

**Comment by JereKoskela** [comment]
I'm happy to put something together, probably from next week, though I don't recall ever hearing about an ancestral gene transfer graph so my review will be demonstrably incomplete. I think most things I'm aware of will be special cases or minor tweaks of the [Donnelly & Kurtz ancestral influence graph](https://projecteuclid.org/journals/annals-of-applied-probability/volume-9/issue-4/Genealogical-processes-for-Fleming-Viot-models-with-selection-and-recombination/10.1214/aoap/1029962866.full), but I can at least elaborate on that.

I think @hyanwong is right about the dual meaning of ARG as the process and its realisation. I think the same is even true of the coalescent, except that it tends to be disambiguated as either coalescent process or e.g. coalescent tree. I wonder if something like "ARG process" would be viable terminology.

**Reply by jeromekelleher** [reply]
Thanks @JereKoskela, that would be great. 

Re the terminology, "ARG process" or "ARG model" sounds good to me. So the coalescent with recombination is an ARG *model*, in that it is a stochastic process that produces ARGs.

**Reply by hyanwong** [reply]
Or "ARG generator" / "ARG generating process"?

**Reply by gregorgorjanc** [reply]
In stats, "model" and "process" (hence also "distribution") are often used interchangeably. Perhaps sticking to "process" is clearer to indicate it spits (generates) something out?
